### DISCUSSION TITLE
Boolean values in dataSource not rendering on UI in Table component

### DISCUSSION THREAD

**Discussion by DanishOnGit** [ant-design/ant-design#41022] [Q&A] [answered]
Hey Everyone! Please Help!

I have a dataSource as the foll:
```
[ 
 {
  correctMovement:false,
  correctObject:false,
  totalLevelClicks:6
  }
]
```

So while rendering this on UI, the totalLevelCLicks are appearing correctly. But all boolean values like correctMovemnet and correctObject are not showing up on UI.

how can I make them appear on UI?
![image](https://user-images.githubusercontent.com/65118743/222437526-5343e5b7-9d0f-4050-adaa-d9d5b92b97a2.png)

**Comment by kiner-tang** [comment, marked as answer]
Because false is a legal child node in React, it will be rendered as an empty node like null.

**Reply by DanishOnGit** [reply]
Okkay got it @kiner-tang . Thank you so much for the quick response! ♥

**Reply by kiner-tang** [reply]
if you want to render `false`, you can use `render` to display it.
- https://ant.design/components/table-cn#column
<img width="1263" alt="image" src="https://user-images.githubusercontent.com/10286961/222442528-515327f9-62cd-459c-899c-b2507b337531.png">

**Reply by kiner-tang** [reply]
like this
```tsx
render(text, record) => `${record.correctMovement}`
```
